### PR TITLE
Update to 4.7 ASU themes from AFS

### PR DESCRIPTION
**Pull request by jslevin4**
Update the latest branch to the 4.7 ASU themes
from AFS.

### CODE REVIEW

File: css/asu_header.css [hunk 1]
```diff
@@ -32,7 +32,7 @@
     display:block;
     overflow:visible!important;
     margin:0;
-    font-size:.8em;
+    font-size:.8rrem;
```
Comment (wmcconnell) [inline css/asu_header.css:35]: I think you might have a typo here, as this was probably meant to be `.8rem`

Comment (jslevin4) [reply css/asu_header.css:35]: Yup. I think it's this way in AFS as well, but the minified css file looks fine.  Hmm.

Comment (jslevin4) [reply css/asu_header.css:35]: Colton merged the changes in, so no need for this.  Closing...

Comment (jslevin4) [reply css/asu_header.css:35]: Walt, thanks for taking a look and finding that typo!

**Pull request closed without merging**